metadata/job.go

```diff
@@ -131,6 +131,28 @@ func (m Metadata) Strings() map[string]string {
 	return bm
 }
 
+/* GetListOfStrings get list of strings if exist, and true if they key is present.
+If value is list of strings we return it as is.
+If value is list of interfaces, we try convert it into list of strings, if fail we return an empty list */
+func (m Metadata) GetListOfStrings(name string) ([]string, bool) {
```

Comment:
Can you pick a name that more closely follows the other methods? Otherwise LGTM!